docs/development/deploying-locally/index.md

```diff
@@ -1,3 +1,50 @@
 # Deploying locally
 
 REANA can be easily deployed locally on your laptop using Minikube.  Useful for REANA platform development.
+
+### Instructions
+
+* (1) Clone the main REANA repository:
+```console
+$ git clone https://github.com/reanahub/reana.git
+$ cd reana
+```
+
+* (2) Create a REANA development setup as follows:
+```console
+$ GITHUB_USER=JOHNDOE make setup clone prefetch
```

Comment:
- `GITHUB_USER=johndoe` lowercase name 

- note that `make clone` won't work for novice users. Two solutions: (i) just install REANA from `upstream` (which can be executed in battch but may accidentally lead to troubles later; (ii) separate "fork" step which requires manual action (since we are not using `hub`) well before launching `make` commands.

(1) Clone...
(2) Fork...
(3) Set up dev...

This would need to `mkvirtualenv reana && pip install . --upgrade` so that people have `reana-dev` command after step (1).